src/radical/pilot/agent/agent_0.py

```diff
@@ -340,77 +350,59 @@ def _start_services(self):
         If a `./services` file exist, reserve a compute node and run that file
         there as bash script.
         '''
+        self._log.info("Starting the agent services")
+        service_descriptions = self._cfg.services
+        services = list()
+
+        for service_desc in service_descriptions:
+            task = dict()
+            task['origin'] = 'services'
+            task['description'] = TaskDescription(service_desc).as_dict()
+            task['state'] = rps.AGENT_STAGING_INPUT_PENDING
+            task['status'] = 'NEW'
+            task['type'] = 'task'
+            task['uid'] = ru.generate_id('services.%(item_counter)04d',
+                                             ru.ID_CUSTOM, ns="TODO_WHATTOUSE")
```

Comment:
`ns` should be the session ID (the uid is supposed to be unique in the scope of this session).

Reply:
fixed